core/src/main/java/bio/terra/pearl/core/service/participant/FamilyEnrolleeService.java

```diff
@@ -20,15 +22,23 @@ public FamilyEnrolleeService(FamilyEnrolleeDao familyEnrolleeDao,
     }
 
     @Transactional
-    public void deleteByFamilyId(UUID familyId) {
-        dao.deleteByFamilyId(familyId);
+    public void deleteByFamilyId(UUID familyId, DataAuditInfo info) {
+        List<FamilyEnrollee> objs = dao.findByFamilyId(familyId);
+
+        bulkDelete(objs, info);
     }
 
     @Transactional
-    public void deleteByEnrolleeId(UUID enrolleeId) {
-        dao.deleteByEnrolleeId(enrolleeId);
-    }
+    public void deleteByStudyEnvironmentId(UUID studyEnvironmentId, DataAuditInfo info) {
```

Comment:
this should definitely not be audited--this only happens in the context of study deletion

Reply:
Before this PR, I noticed that there was a delete happening at the study env level that just did:
```
        enrolleeRelationService.deleteByStudyEnvironmentId(studyEnvironmentId, null);
```
I think I slightly prefer having every delete be audit-able but passing null in the specific cases where we don't want it. That way we don't accidentally use an unaudited function without thinking about it. What do you think?

Reply:
hmmm... I think I like that the deleteByStudyEnvId doesn't take an audit context, since that's how it's communicating that we don't want the caller to audit that operation.   Agree that there's some danger of copy-pasting that method and creating a new one that should be audited, but isn't.  But I think that's better than getting in the habit of passing `null` to audit infos.  

Awhile ago, I toyed with the idea of having separate services for the "illegal/unaudited" operations.  So e.g. there would be a separate "EnrolleeRelationUnauditedService" that would contain the deleteByStudyEnvId method.  But I couldn't get the ergonomics to work out -- downstream services ended up needing to include both the unaudited and audited versions, and it was a dependency mess.  But something like that might be worth revisiting

Reply:
per discussion, i refactored all remaining deletebystudyenv methods to not take in dataauditinfo